### PR TITLE
Update Sitemap Page Design

### LINKED ISSUE #1049: Create a sitemap
### What's your idea?

We have a lot of pages that are not included in our navbar. We should provide a sitemap for members to know what pages we have!

References:
Link: https://developers.google.com/search/docs/crawling-indexing/sitemaps/overview
image

### PR DESCRIPTION
**Pull request by 26samaahmed**
Fix #1049 

Screenshot 2024-04-21 at 1 58 03 PM
Screenshot 2024-04-21 at 1 58 34 PM

### CODE REVIEW

File: src/lib/public/sitemap/sitemap.ts [hunk 1]
```diff
@@ -0,0 +1,9 @@
+export interface SiteMapEntry {
+  id: string;
+  title: string;
+  link: string;
+}
+
+export type SiteMap = {
```
Comment (EthanThatOneKid) [inline src/lib/public/sitemap/sitemap.ts]: Run `npm run all` to automatically resolve any formatting errors.

**Review by EthanThatOneKid — CHANGES_REQUESTED** [review]

File: src/lib/public/sitemap/sitemap.ts (hunk 1 above)
Comment (EthanThatOneKid) [reply src/lib/public/sitemap/sitemap.ts]: Resolved in <https://github.com/EthanThatOneKid/acmcsuf.com/pull/1069/commits/a51a7c94da5aef6b80cdedc39e86ef9147876a12>.

**Review by EthanThatOneKid — APPROVED** [review]
LGTM, @26samaahmed!

**Comment by EthanThatOneKid** [comment]
This looks gr8!

**Comment by k4rni** [comment]
Thank you @26samaahmed and @tylermlui for helping on this page!

**Pull request merged**